### PR TITLE
Clarify docs headers

### LINKED ISSUE #427: Docs: Clarification that file schema means top-level field's tag schemas
In the docs there's a handful of mentions of "file schema" which are separately listed than the "tags". For those unfamiliar with ASDF, this isn't particularly harmful. For those more familiar with ASDF, it's a bit ambiguous because file schema can also mean a schema to describe the file schema as a whole.

Discussing with @braingram , I come to learn that ...

> the "file schema" headings there refer to the ASDF schema used for the top-level roman object in the file.
> — @braingram

..., therefore all file schemas are actually tag schemas and only validates the fields under `roman` rather than a schema for the entire file. 

```yaml
...
roman: !<asdf://path/to/schemas/schema-x.y.z>
  meta: ...
  data: ...
```

Furthermore, there isn't a true file schema that enforces `roman` must be used as the top level. Perhaps something with this schema...

```yaml
allOf:
  - $ref: http://stsci.edu/schemas/asdf/asdf-schema-1.0.0
  - type: object
    properties:
      roman:
        type: object
    required: [roman]
```

### PR DESCRIPTION
**Pull request by braingram**
Updates schema docs to:
- remove mention of "File schemas"
- add a one sentence description
- add links to asdf schema docs and roman_datamodels docs

Link to updated docs: https://rad--446.org.readthedocs.build/en/446/schemas.html

Only a docs change so no downstream tests will be run.

Closes #427

## Tasks
- [ ] Update or add relevant `rad` tests.
- [x] Update relevant docstrings and / or `docs/` page.
- [ ] Does this PR change any schema files?
  - [ ] Schema changes were discussed at RAD Review Board meeting.
- [ ] Does this PR change any API used downstream? (If not, label with `no-changelog-entry-needed`.)
  - [x] Write news fragment(s) in `changes/`: `echo "changed something" > changes/<PR#>.<changetype>.rst` (see below for change types).
  - [ ] Start a `romancal` regression test (https://github.com/spacetelescope/RegressionTests/actions/workflows/romancal.yml) with this branch installed (`"git+https://github.com/<fork>/rad@<branch>"`).
  - [ ] Update relevant `roman_datamodels` utilities and tests.

<details><summary>News fragment change types:</summary>

- ``changes/<PR#>.feature.rst``: new feature
- ``changes/<PR#>.bugfix.rst``: fixes an issue
- ``changes/<PR#>.doc.rst``: documentation change
- ``changes/<PR#>.removal.rst``: deprecation or removal of public API
- ``changes/<PR#>.misc.rst``: infrastructure or miscellaneous change
</details

### CODE REVIEW

**Comment by braingram** [comment]
@PaulHuwe I don't appear to have permissions to request reviewers. Would you take a look?

Also @ketozhang how does the updated wording look to you?

File: docs/schemas.rst [hunk 1]
```diff
@@ -3,29 +3,33 @@
...
+Level 3 (resampled mosaic) schema
+---------------------------------
 
 .. asdf-autoschemas::
 
   wfi_mosaic-1.0.0
 
-Level 4 File schemas
---------------------
+Level 4 schemas
```
Comment (PaulHuwe) [inline docs/schemas.rst:31]:
```suggestion
Level 4 (ancillary) schemas
```
Just a suggestion to match the descriptions of the other levels?

**Review by PaulHuwe — APPROVED** [review]
LGTM with one suggestion

File: docs/schemas.rst (hunk 1 above)
Comment (braingram) [reply docs/schemas.rst]: Thanks! I applied the suggestion and fixed the header marks to match in https://github.com/spacetelescope/rad/pull/446/commits/7ce376ee1cfad9c9bf8fa5a38d8a849f412cbc31

**Comment by ketozhang** [comment]
Looks better, thank you @braingram 

Couple of 'file schema' left in the sidebar. If these are truly schemas for the entire file, then this makes sense.
image

Looks like something is wrong with the TOC
image

**Comment by braingram** [comment]
Thanks for taking a look.

https://github.com/spacetelescope/rad/pull/447 addresses the TOC

As the `Reference File Schemas` are schemas used in reference files I think we have to live with that title. I made a slight change to the page to try and remove some of the ambiguity.

I updated the `Ground Tests File Schemas` to `Ground Tests Schemas`.

The issue will remain searchable after it's closed which can also be helpful if someone else runs into this confusion.

**Pull request merged**